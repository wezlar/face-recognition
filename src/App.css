.App {
  text-align: center;
}

.App-header {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: #333;
}

.App-link {
  color: red;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
