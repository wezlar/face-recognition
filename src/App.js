import React from 'react';
import './App.css';

import Camera from './components/camera/Camera';

function App() {
  return (
    <div className='App'>
      <header className='App-header'>
        <p>
          Test app
        </p>
        <Camera />
      </header>
    </div>
  );
}

export default App;
