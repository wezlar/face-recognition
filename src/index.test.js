import React, { Component } from 'react';
import {
  cleanup,
} from '@testing-library/react';
import ReactDOM from 'react-dom';
import App from './App';
jest.mock('./components/VideoInput/VideoInput', () => 'VideoInput');
jest.mock('./components/ImageInput/ImageInput', () => 'ImageInput');
jest.mock('./components/Navigation/Navigation', () => 'VideoInput');

jest.mock('react-dom', () => ({ render: jest.fn() }));
afterEach(cleanup);

describe('Application root', () => {
  test('should render without crashing', () => {
    const div = document.createElement('div');
    div.id = 'root';
    document.body.appendChild(div);
    require('./index.js');
    expect(ReactDOM.render).toHaveBeenCalled();
    expect(ReactDOM.render).toMatchSnapshot();
  });
});
