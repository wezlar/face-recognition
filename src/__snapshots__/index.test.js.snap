// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application root should render without crashing 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      <BrowserRouter>
        <div>
          <VideoInput
            links={
              Array [
                Object {
                  "label": "Home",
                  "url": "/",
                },
                Object {
                  "label": "Video Input",
                  "url": "/video",
                },
                Object {
                  "label": "Image",
                  "url": "/image",
                },
              ]
            }
          />
          <Route
            component={[Function]}
            exact={true}
            path="/"
          />
          <Route
            component="VideoInput"
            path="/video"
          />
          <Route
            component="ImageInput"
            path="/image"
          />
        </div>
      </BrowserRouter>,
      <div
        id="root"
      />,
    ],
  ],
  "results": Array [
    Object {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
